.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think that it will be better to remove IDE configuration from our git repositories and add IDE default directories to the gitignore to prevent future commits with these settings